abi3audit/_audit.py

```diff
@@ -80,11 +80,11 @@ def __rich_console__(self, console: Console, options: ConsoleOptions) -> RenderR
             yield f"[green]:thumbs_up: {self.so}"
 
 
-def audit(so: SharedObject) -> AuditResult:
+def audit(so: SharedObject, assume_minimum_abi3: Tuple[int, int] = (3, 2)) -> AuditResult:
```

Comment:
Rather than a tuple, could we make this a `abi3info.models.PyVersion`?

There's a `parse_dotted` helper we could reuse as well, for pulling the version from the CLI: <https://woodruffw.github.io/abi3info/models.html#abi3info.models.PyVersion.parse_dotted>

Reply:
Nice! I am using it now, but it is less strict that the CLI checks, so we have a mixed logic. Maybe worth upstreaming to `abi3info`?

Reply:
Yep! An upstream PR would be appreciated.